### PR TITLE
feat: added text overrides on CartItemDetail

### PR DESCRIPTION
**Pull request by HarisSpahijaPon**
Part of: #409 
Impact: **minor**  
Type: **feature**

## Component
CartItemDetail now has overrides when i18next will be implemented (see #409)

## Breaking changes
No breaking chances, all added fields are optional

## Testing
1. Add a new prop `quantityText` to `<CartItemDetail />` 
2. Add the value `"test"` to `quantityText`
3. Label should display "test" when test is given as a value to the prop. When the prop `quantityText` is not added, it should default to the values given in default props

### CODE REVIEW

File: package/src/components/CartItemDetail/v1/CartItemDetail.js [hunk 1]
```diff
@@ -79,12 +79,20 @@ class CartItemDetail extends Component {
...
+     * The text for the "Quantity" label text.
+     */
+    quantityText: PropTypes.string,
     /**
      * Product title of chosen item.
      */
     title: PropTypes.string
   };
 
+  static defaultProps: {
```
Comment (HarisSpahijaPon) [inline package/src/components/CartItemDetail/v1/CartItemDetail.js]: has to be "=" instead of :

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit 02017c567cdcf90d23eab9dfd864ae1ba6ab4f3f

https://deploy-preview-450--stoic-hodgkin-c0179e.netlify.com

**Comment by HarisSpahijaPon** [comment]
done

**Pull request merged**